Source/CheatSheet/CQLBasics-cheatsheet.md

```diff
@@ -0,0 +1,121 @@
+# CQL Basics Cheat Sheet
+
+## Values
+
+|Value|Description|Example|
+|-----|-----------|-------|
+|Null|The null literal|<pre lang="cql">null</pre>|
+|Boolean|The boolean literals|<pre lang="cql">true, false</pre>|
+|Integer|Sequences of digits in the range 0..2<sup>31</sup>-1|<pre lang="cql">16, -28</pre>|
+|Long|Sequences of digits in the range 0..2<sup>63</sup>-1|<pre lang="cql">16000000000, -28000000000</pre>|
+|Decimal|Sequences of digits with a decimal point, in the range 0.0.. (10<sup>28</sup>-1)/10<sup>8</sup>|<pre lang="cql">100.015</pre>|
+|String|Strings of any character enclosed within single-ticks (')|<pre lang="cql">'pending', 'active', 'complete'</pre>|
+|Date|The at-symbol (@) followed by an ISO-8601 compliant representation of a date|<pre lang="cql">@2014-01-25</pre>|
+|DateTime|The at-symbol (@) followed by an ISO-8601 compliant representation of a datetime|<pre lang="cql">@2014-01-25T14:30:14.559&#13;@2014-01T</pre>|
```

Comment:
Is `@2014-01T` valid?

Reply:
Yes. I took the examples directly from the spec: https://cql.hl7.org/02-authorsguide.html#simple-values